Counties/Florida/Bay County/Scraper/utils/ScraperUtils.py

```diff
@@ -61,40 +61,78 @@ def parse_plea_case_numbers(plea_text: str, valid_charges: List[int]) -> List[in
     :param valid_charges: Valid charge numbers that can be referenced in the plea text
     :return: List of charge count numbers related to this plea
     """
-    plea_text = plea_text.strip().split()
-    if len(plea_text) > 0:
-        plea_counts = re.sub('[^,0-9]', '', plea_text[-1]).split(',')
-        plea_number = [int(charge) for charge in plea_counts if charge != '']
-        # In rare cases numbers appear in the plea text not relating to a count number. Filter these out.
-        filtered_plea_number = [charge for charge in plea_number if charge in valid_charges]
-        return filtered_plea_number
-    else:
+    try:
```

Comment:
I think it should be possible to get rid of this try/except with `plea_text = plea_text or ''`. As a general note, nothing seems to be enforcing our type annotations and we should probably fix that. It would prevent things like None being passed in as a `str` (a None-able `str` is an `Optional[str]` from my understanding).

Reply:
In Python enforced typing is not something typically done, instead type annotations are treated as hints to the IDE, so will cause an IDE warning that you've passed in the wrong type (if the IDE can work out the type).

In [Python documentation](https://docs.python.org/3/library/typing.html), they are referred to as 'type hints', rather than any sort of enforced type system.

Python's design philosophy is often given the slogan: "We're all consenting adults here", meaning you are expected to be aware of what you can pass to a function and what you can't. It's the same reason Python classes don't have private member variables.

Enforced typing is technically possible with some libraries, but I suggest we avoid that as it's not normal python practice.

Reply:
I don't think we should enforce this at runtime, but a I can see value in a linter enforcing this. Regardless, not in-scope for this PR.